clang/lib/CodeGen/CGExpr.cpp

```diff
@@ -1989,6 +1989,14 @@ llvm::Value *CodeGenFunction::EmitLoadOfScalar(Address Addr, bool Volatile,
     return EmitAtomicLoad(AtomicLValue, Loc).getScalarVal();
   }
 
+  if (const auto *BIT = Ty->getAs<BitIntType>()) {
+    if (BIT->getNumBits() > 128) {
```

Comment:
For a number of bits >64, <=128, LLVM's `iN` type will have identical representation to Clang `_BitInt(N)` but different alignment. I think this is fine, I think nothing needs their alignment to match Clang's, but could you double-check to make sure you agree?

Reply:
These types remain unchanged.